.github/workflows/publish.yml

```diff
@@ -30,3 +24,14 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.CI_GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.CI_GITHUB_TOKEN }}
           NODE_AUTH_TOKEN: ${{secrets.CI_GITHUB_TOKEN}}
+
+      ## Lets update the package.json and then publish with a different name to npm registry.
+      - run: echo $(jq '.name = "@pocket-tools/terraform-modules" | del(.publishConfig)' package.json) > package.json
```

Comment:
We dont want to do semantic release twice. Instead we want to modify the output and then do a npm publish